modules/tinymce/src/core/main/ts/api/dom/EventUtils.ts

```diff
@@ -209,7 +212,9 @@ const bindOnReady = function (win, callback, eventUtils) {
   }
 
   // Fallback if any of the above methods should fail for some odd reason
-  addEvent(win, 'load', readyHandler);
+  if (!eventUtils.domLoaded) {
```

Comment:
This additional check was needed, as on modern browsers the above `DOMContentLoaded` callback fires immediately.